python/_fio_plugin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip git make gcc fio
+RUN mkdir /plugin
+COPY requirements.txt /plugin
+RUN pip3 install --requirement /plugin/requirements.txt
+COPY fio_plugin.py /plugin
+COPY fio_schema.py /plugin
+
+WORKDIR /plugin
+# USER 1000
+ENTRYPOINT ["python3.9", "fio_plugin.py", "-f", "/plugin/fio-job.yaml", "--debug" ]
```

Comment:
Your entrypoint should be the plugin without any flags. The flags can be passed to the container when run directly, and we will soon have support for the plugin to accept stdin for the input params.